dev/dll/DensityStyles/Compact.xaml

```diff
@@ -17,5 +17,8 @@
     <Thickness x:Key="ComboBoxEditableTextPadding">10,0,30,0</Thickness>
     <x:Double x:Key="ComboBoxMinHeight">24</x:Double>
     <Thickness x:Key="ComboBoxPadding">12,1,0,3</Thickness>
+    <Thickness x:Key="ToggleSwitchTopHeaderMargin">0,0,0,0</Thickness>
+    <x:Double x:Key="ToggleSwitchPreContentMargin">6</x:Double>
+    <x:Double x:Key="ToggleSwitchPostContentMargin">6</x:Double>
```

Comment:
Do these styles override theose defined in ToggleSwithc_themeresources? If so, why do the compact density tests still fail?

Reply:
@StephenLPeters @licanhua any ideas ?

Reply:
I'm not 100% sure about what I described below since it's been about 2 years.

yes. compact overrides ToggleSwitch_themeresources.
Since RS5, we introduced density and compact, We have the old standard, new standard and compact.
Here density means the new standard the UI size in RS5.

We can think RS4 or before UI design is old standard,
and RS5 is new standard (smaller)
and Compact is for enterprise customer(smallest) because they have complex forms and want to put more content in the same page. 

Let me make ToggleSwitchTopHeaderMargin as an example:
RS4 UI design: ToggleSwitchTopHeaderMargin = 0,0,0,4
RS5 UI design: ToggleSwitchTopHeaderMargin = 0,0,0,0
RS5 UI compact design: ToggleSwitchTopHeaderMargin = 0,0,0,0

Here density testing only means the RS5 UI design. so it's complete compact unrelated. Any change on compact.xaml should not impact the density test case.

If you are sure this margin change is expected, please change the code
```
string expectedHeaderMargin = "0,0,0,0";
```

Reply:
Oh hmm I see. Thank you @licanhua ! Updated the test correctly now.